tests/testthat.R

```diff
@@ -1,4 +1,9 @@
+# TODO. Not sure why this is required
+library(rClr)

```

Comment:
I don't understand WY I need to load the rCLR since it's a dependencies from the package. Any idea?